lib/assignDisabledRanges.js

```diff
@@ -53,10 +53,53 @@ module.exports = function (root, result) {
 	};
 
 	result.stylelint.disabledRanges = disabledRanges;
-	root.walkComments(checkComment);
+
+	// Work around postcss/postcss-scss#109 by merging adjacent `//` comments
+	// into a single node before passing to `checkComment`.
+
+	/** @type {PostcssComment?} */
+	let inlineEnd;
+
+	root.walkComments((/** @type {PostcssComment} */ comment) => {
+		if (inlineEnd) {
+			// Ignore comments already processed by grouping with a previous one.
+			if (inlineEnd === comment) inlineEnd = null;
+		} else if (isInlineComment(comment)) {
+			const fullComment = comment.clone();
+			let next = comment.next();
+
+			while (next && next.type === 'comment') {
+				/** @type {PostcssComment} */
+				const current = next;
+
+				if (!isInlineComment(current)) break;
+
+				fullComment.text += `\n${current.text}`;
+
+				if (fullComment.source && current.source) {
+					fullComment.source.end = current.source.end;
+				}
+
+				inlineEnd = current;
+				next = current.next();
+			}
+			checkComment(fullComment);
+		} else {
+			checkComment(comment);
```

Comment:
what do you think about adding second argument `checkComment(comment: Comment, text: string| string[])` this prevents of extra allocations using 
```js
fullComment.text += `\n${current.text}`
```
in checkComment check first one `COMMAND_PREFIX` found

Reply:
Would putting all the texts into an array and then joining them and assigning the result to `fullComment.text` before calling `checkComment` work? Passing it separately could be confusing, since the text in the parameter and the text of the comment wouldn't always match (and it would require adding that text parameter to all the `process*Command` functions).

Reply:
`+=` is pretty highly-optimized in V8... per [this benchmark page](https://web.archive.org/web/20181117202141/https://jsperf.com/join-concat/2), it seems to actually be faster than `.join()`.

Reply:
lets keep `+=`, as I understand it affects only inline comments..